packages/babel-compiler/babel-compiler.js

```diff
@@ -194,7 +157,7 @@ BCp.initializeMeteorAppSwcrc = function () {
 let lastModifiedSwcLegacyConfig;
 BCp.initializeMeteorAppLegacyConfig = function () {
   const swcLegacyConfig = convertBabelTargetsForSwc(Babel.getMinimumModernBrowserVersions());
-  if (lastModifiedMeteorConfig?.modern?.transpiler?.verbose && !lastModifiedSwcLegacyConfig) {
+  if ((lastModifiedMeteorConfig?.modern?.transpiler?.verbose || this.extraFeatures?.verbose) && !lastModifiedSwcLegacyConfig) {
```

Comment:
If I'm right, I saw this condition `(lastModifiedMeteorConfig?.modern?.transpiler?.verbose || this.extraFeatures?.verbose)`  three times. Could you create a variable for it, please?

Reply:
Done